build.gradle

```diff
@@ -63,6 +63,7 @@ dependencies {
     implementation group: 'org.openjfx', name: 'javafx-web', version: javaFxVersion, classifier: 'win'
     implementation group: 'org.openjfx', name: 'javafx-web', version: javaFxVersion, classifier: 'mac'
     implementation group: 'org.openjfx', name: 'javafx-web', version: javaFxVersion, classifier: 'linux'
+    implementation 'org.controlsfx:controlsfx:11.0.1-SNAPSHOT'
```

Comment:
Could you add a comment here so we know why this is included?

Reply:
ok